src/applications/find-va-forms/containers/SearchResults.jsx

```diff
@@ -14,11 +14,11 @@ const ASCENDING = 'ASC';
 const DESCENDING = 'DESC';
 const FIELD_LABELS = [
   {
-    label: 'VA form number',
+    label: 'Form #',
```

Comment:
Both of these column header title changes need to be approved by design I think! :)